charts/eks-anywhere-packages/templates/awssecret.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if not .Values.workloadOnly }}
```

Comment:
Keep the -not workload only- for now. You'd have to change the namespace otherwise. We'll only support 1 set of creds for now.